src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -357,13 +357,14 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    * specified @ref trait_set, and which traits it is capable of
    * resolving or persisting data for.
    *
-   * This method is usually called early on by a host to determine
-   * whether to enable OpenAssetIO related functionality when handling
-   * specific kinds of data. The host will often adapt its subsequent
-   * behaviour to minimise unsupported interactions with the manager. In
-   * high call volume scenarios (such as CG rendering), this can
-   * potentially save hundreds of thousands of redundant calls into the
-   * API for unmanaged entity traits.
+   * A manager's policy must be runtime invariant for any given set of
+   * inputs. This method is usually called early on by a host to
```

Comment:
[Consider] I think this is an innapropriate place for this, it's taking the oxygen away from the description of how management policy is used.

I'd recommend either adding it below in it's own paragraph (or as a `@note`), or appending it to the end of the initial paragraph, which is more definitional already.

Reply:
Hows [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/1120/commits/748bb5445f1c5e26472091159050853b770ce1f4)?